ext/simplexml/simplexml.c

```diff
@@ -514,8 +514,7 @@ static zval *sxe_prop_dim_write(zend_object *object, zval *member, zval *value,
 			case IS_OBJECT:
 				if (Z_OBJCE_P(value) == sxe_class_entry) {
 					if (sxe_object_cast_ex(Z_OBJ_P(value), &zval_copy, IS_STRING) == FAILURE) {
-						zend_error(E_ERROR, "Unable to cast node to string");
```

Comment:
We can do this change right? I would also a test for it too, but I didn't manage to triggger it as well.

Reply:
Yes, this is fine, but we should also return after the throw.

Reply:
It looks like this is a repeating error for me today :/